tensorboard/webapp/persistent_settings/_data_source/types.ts

```diff
@@ -19,6 +19,20 @@ export enum ThemeValue {
   DARK = 'dark',
 }
 
+// When adding a new value to the enum, please implement the deserializer on
+// data_source/metrics_data_source.ts.
+// When editing a value of the enum, please write a backward compatible
+// deserializer in tensorboard/webapp/metrics/store/metrics_reducers.ts
+export enum TooltipSort {
```

Comment:
This is not a "persistent settings" type it is a "metrics" type. Can we continue to model it that way?

You probably felt the need to put this here because keeping it in metrics would introduce a circular dependency. Can we instead move this to metrics/types (note, not metrics/internal_types) and unwind the circular dependency?

The answer might be to change teh dependency of "metrics/stores:types" from "metrics/data_source" to "metrics/data_source:types" and change metrics_types.ts to get the symbols from '../data_source/types'.

Reply:
Actually I don't think that the persistent settings needs a dependency on the metrics/store:types anymore so this can be resolved fairly easily.